sites/kit.svelte.dev/src/routes/index.svelte

```diff
@@ -73,8 +117,7 @@
 		</p>
 
 		<p>
-			Read the <a href="https://svelte.dev/blog/whats-the-deal-with-sveltekit">introductory blog</a>
-			post to learn more.
+			<a href="https://stackblitz.com/fork/svelte-kit">Try on StackBlitz</a> or create a project locally.
```

Comment:
node.new/sveltekit instead?

Reply:
done. what's that URL though? I don't know who owns it...

Reply:
official launcher by stackblitz afaik. https://mobile.twitter.com/Rich_Harris/status/1461876128474447875